docs/source/manifest_submission.rst

```diff
@@ -0,0 +1,274 @@
+Submit a manifest to Synapse
+============================
+
+Prerequisites
+-------------
+
+**Obtain Synapse Credentials**:
+Ensure you have a Synapse account and have obtained your Synapse credential by following the instructions here:
+    `<https://python-docs.synapse.org/tutorials/authentication/#prerequisites>`_
+
+**Before Using the Schematic CLI**
+
+- **Install and Configure Schematic**:
+  Ensure you have installed `schematic` and set up its dependencies.
+  Refer to the **"Installation Guide for Users"** for detailed instructions.
+
+- **Understand Important Concepts**:
+  Familiarize yourself with key concepts outlined on the :doc:`index.rst` page.
+
+- **Configuration File**:
+  Learn more about each attribute in the configuration file by referring to the relevant documentation.
+
```

Comment:
Is there somewhere we can link to for this?

Reply:
link was added.